### DISCUSSION TITLE
without the "radicale" in URL?

### DISCUSSION THREAD

**Discussion by charleslcso** [Kozea/Radicale#1568] [Q&A reverse proxy]
I got Nginx reverse proxy working. 

The web UI URL is https://ds/radicale/.web/. 

The collection URL is https://ds/radicale/charles/test/

Is it possible to set up Nginx to take out the word "radicale"?

I've tried various Nginx conf changes, and all changes will give me 404 from Radicale.

Thanks a lot!

**Comment by pbiering** [comment]
I've added now a example file to repository, see here for example: https://github.com/Kozea/Radicale/blob/master/contrib/nginx/radicale.conf, it looks ike working by adjusting to `location /`.
